protos/flow.proto

```diff
@@ -32,16 +31,16 @@ message TableMapping {
 }
 
 message SetupInput {
-  peerdb_peers.Peer peer = 1;
+  string peer = 1;
```

Comment:
what is the upgrade story for this, also if we're going to break protos might as well split it off and do other changes as well

Reply:
hoping pause unpause will make it through alright, but have go ahead to make breaking changes for this. Could deprecate fields & add name fields & fallback to name from deprecated config